Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -598,25 +598,29 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
                 innerElement.append(item);
         }
 
-        let swatch = new WI.InlineSwatch(type, valueObject, {readOnly: !this._isEditable()});
+        let colorMapping = this.property.ownerStyle.nodeStyles.computedStyle.variablesForType(WI.CSSStyleDeclaration.VariablesGroupType.Colors);
+        let colorPalettes = [];
+        for (let i = 0; i < colorMapping.length; i++) {
+            if (colorMapping[i].name == this.property.name) {
+                continue;
+            }
+
+            colorPalettes.push({
+                color: WI.Color.fromString(colorMapping[i].value),
+                colorSource: colorMapping[i].name
```

Comment:
Planning on doing this with the next patch that will introduce the colorPalette object that will do all of this internally, for the initial feature because it's only using the source from the computed styles section, I choose to omit it